client/coral-embed-stream/src/tabs/profile/components/Comment.js

```diff
@@ -76,7 +76,6 @@ class Comment extends React.Component {
           <div className="my-comment-asset">
             <a
               className={cn(styles.assetURL, 'my-comment-anchor')}
-              href="#"
               onClick={this.goToStory}
```

Comment:
Was there a reason @okbel why we used an `onClick` instead of just passing the `href` the asset url?